### ISSUE TITLE
Communication problems

### ISSUE THREAD

**Issue by unleu** [tudorsss/Food-Tracker#68]
1. User should be able to distinguish which message is from which conversation. The way they are now it is impossible to know which message is the reply of which message. 
2.

**Comment by tudorsss** [comment]
Hey @unleu,

You should have the sender header & then the sent to footer in each of the messages. I'm assuming the admin user should see all messages, but the rest of those users should see only those messages in which they got "tagged". Isn't this how it works right now?

**Comment by unleu** [comment]
Yes, but if there is a conversation between user A and user B about patient C and another conversation between user A and user B about patient C, how can you distinguish them as an admin.

Or, if there is a conversation between user A and user B about patient C regarding treatment D and another one between user A and user B about patient C regarding treatment E how do they distinguish them? what  if they are simultaneous ?

**Comment by tudorsss** [comment]
Hey @unleu, I'm seeing that these are the limitations of the current messaging system; we would require to implement subjects and message threading; I am thinking that this is more of a bug than a feature; what do you want to do with this ticket? 

Feature reporting is not something we would need to think about at the moment.

**Comment by unleu** [comment]
It's more of a feature.. but it could be a bug as well. you decide :)

**Comment by tudorsss** [comment]
Well @unleu, if you find a way to report this as a bug, we'll add it in Youtrack and see where that takes us.
